IntegrationTests/UITests/CommandsDialogs/FormBrowseTests.LeftPanel.ReorderNodes.cs

```diff
@@ -56,25 +51,32 @@ public void OneTimeTearDown()
         [SetUp]
         public void SetUp()
         {
+            // We don't want avatars during tests, otherwise we will be attempting to download them from gravatar...
```

Comment:
💡 A better way to handle this is having a global (assembly-wide) "before test" method that initializes `AppSettings` to a known default value. This way any test can make changes to this without worrying about interfering with other tests.

Reply:
A better way to handle this would be convert `AppSettings` from a static to an instance-based type

Reply:
(I have not touched this file. It was only moved by Igor in https://github.com/gitextensions/gitextensions/commit/b196ab8eeb65e8f688c6dda8ceb99eb7439acc2c.)

Reply:
We can tackle it separately.